risc0/zkvm/src/lib.rs

```diff
@@ -47,9 +47,12 @@ pub use risc0_zkvm_platform::{declare_syscall, memory::MEM_SIZE, PAGE_SIZE};
 
 #[cfg(feature = "binfmt")]
 pub use self::binfmt::{elf::Program, image::MemoryImage};
+#[cfg(feature = "profiler")]
+pub use self::exec::profiler::Profiler;
 pub use self::receipt::{SegmentReceipt, SessionReceipt};
 #[cfg(feature = "prove")]
 pub use self::{
+    exec::io::{Syscall, SyscallContext},
```

Comment:
yay!

Reply:
MultiTest doesn't really test the external API, it really just tests the host/guest interaction.

Reply:
I'm also not super keen on exposing the `Syscall` interface directly, I think there are usually better ways to extend the host/guest (like using I/O). If we can drop this re-export I'd prefer to.

Reply:
I am also happy with that.